WeakAurasOptions/RegionOptions/DynamicGroup.lua

```diff
@@ -281,11 +273,55 @@ local function createOptions(id, data)
         WeakAuras.ResetMoverSizer();
       end
     },
-    spacer = {
+    -- border/background options
+    -- TODO: use Weakauras.BorderOptions for these instead
```

Comment:
is that something you want to do this PR or in the future?

Reply:
No, I don't want to do it in this pull request.